frontend/src/views/web/featured-challenge/leaderboard.html

```diff
@@ -41,18 +41,50 @@
+                                                            <i class="fa fa-sort-desc fa-stack-1x" ng-class="!featured_challenge.reverseSort && featured_challenge.sortColumn == 'rank'? 'text-dark-black' : 'text-light-black w-500'"></i>
+                                                    </span>
+                                                </a>
+                                            </td>
+                                            <td class="align-left" data-field="team">
+                                                <a href="#" ng-click="featured_challenge.sortFeaturedLeaderboard(featured_challenge, 'string');">
+                                                    Participant Team
+                                                    <span class="fa-stack fa-1x">
+                                                        <i class="fa fa-sort-asc fa-stack-1x" ng-class="featured_challenge.reverseSort && featured_challenge.sortColumn == 'string'? 'text-dark-black' : 'text-light-black w-500'"></i>
+                                                        <i class="fa fa-sort-desc fa-stack-1x" ng-class="!featured_challenge.reverseSort && featured_challenge.sortColumn == 'string'? 'text-dark-black' : 'text-light-black w-500'"></i>
+                                                    </span>
+                                                </a>
+                                            </td>
+                                            <td class="align-right" ng-repeat="key in featured_challenge.leaderboard[0].leaderboard__schema.labels">
+                                                <a href="#" ng-click="$parent.featured_challenge.sortFeaturedLeaderboard($parent.featured_challenge, 'number', $index);">
+                                                    {{key}}
+                                                    <span class="fa-stack fa-1x">
+                                                        <i class="fa fa-sort-asc fa-stack-1x" ng-class="$parent.featured_challenge.reverseSort && $parent.featured_challenge.sortColumn == 'number' && $parent.featured_challenge.columnIndexSort == $index ? 'text-dark-black' : 'text-light-black w-500'"></i>
+                                                        <i class="fa fa-sort-desc fa-stack-1x" ng-class="!$parent.featured_challenge.reverseSort && $parent.featured_challenge.sortColumn == 'number' && $parent.featured_challenge.columnIndexSort == $index ? 'text-dark-black' : 'text-light-black w-500'"></i>
+                                                    </span>
+                                                </a>
+                                            </td>
+                                            <td class="align-right" data-field="submission_time">
```

Comment:
All your alignments are towards the right or left? Just refer to normal challenge leaderboard.html and make all centre aligned.
```suggestion
                                            <td class="align-center" data-field="submission_time">
```